plotly/tests/test_optional/test_offline/test_offline.py

```diff
@@ -9,6 +9,24 @@
 
 import plotly
 
+# TODO: matplotlib-build-wip
```

Comment:
Can we make an issue for this `TODO` so we can track it in GH and tag this bit of code with the issue number?